src/docs/components/docs-topbar/docs-topbar.css

```diff
@@ -3,23 +3,32 @@
 
   &::part(container) {
     margin: 0 var(--ld-sp-4);
-    padding: 0 var(--ld-sp-24);
+    padding: 0 var(--ld-sp-32);
```

Comment:
Wouldn't it be easier to simply add the breakpoint media queries here, so you don't have to work with negative margins?

Reply:
I don't know. I'll leave it as it is.